### ISSUE TITLE
Напиши нормальный README, ебл@н

### ISSUE THREAD

**Issue by diveloper53** [SashaTalkStudio/SayOnlineWebLite#4]
Читать больно

Это продакшен, а не пет проект какой-то, ъуъ

**Comment by SashaTalk** [comment]
если у тебя больше двух хромосом, то, думаю, ты догадаешься, что репозиторий нужно склонировать, поместить в корень сайта. вебсервер, разумеется должен поддерживать php 8.0-8.2.

оставлю открытым, напишу, может инструкцию по установке

**Comment by diveloper53** [comment]
А
А я думал он на пхп 7.4 работает

**Comment by Tolinme** [comment]
Выключи бота Саша еблан

**Comment by diveloper53** [comment]
Не то слово)

**Comment by SashaTalk** [comment]
> Выключи бота Саша еблан

пусть работает, я че, зря писал?

**Comment by diveloper53** [comment]
Возьми в пример документацию к anonhost.ga.
Там всё подробно расписано, не то что тут.
Хотя тоже, казалось бы, IQ выше чем у хлебушка обладать не надо что разобраться.

**Comment by SashaTalk** [comment]
> Хотя тоже, казалось бы, IQ выше чем у хлебушка обладать не надо что разобраться.

ну вот и я о том же. тем более апи сейчас стабильно продуктивно лежит

**Comment by kaurcev** [comment]
Это API?
Sasha, ну глазам больно :(

**Comment by SashaTalk** [comment]
нет, это не API. это клиент Say Online, который я написал, чтобы зайти с IE4. люди очень-очень просили открыть исходники, что я и сделал.
